app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -181,6 +186,8 @@ class LuciBuildService {
       'git_ref': 'refs/pull/${pullRequest.number}/head',
     });
 
+    String cipdVersion = 'refs/heads/${pullRequest.base!.ref!}';
+    cipdVersion = branches != null && branches.contains(cipdVersion) ? cipdVersion : 'refs/heads/main';
```

Comment:
Instead of hardcoding, can we define the default cipdVersion in config to reference?

Reply:
done